Runtime/CourseController/StandaloneCourseController.cs

```diff
@@ -459,7 +459,13 @@ private void SetupLanguagePicker()
             });
             
             // If there is only one option, the dropdown is currently disabled.
-            languagePicker.enabled = supportedLanguages.Count > 1;
+            if (supportedLanguages.Count > 1 == false)
```

Comment:
```suggestion
            if (supportedLanguages.Count < 2)
```
Seems more readable to me. But I am also fine without the change.